README.md

```diff
@@ -15,6 +15,9 @@ Supports any project with package managers that install npm dependencies into a
 
 <img src="https://github.com/sonatype-nexus-community/auditjs/blob/alpha/assets/images/auditjs.png?raw=true" width="640">
 
+## Requirement
+For users that are hoping to scan with Nexus IQ Server, Version 77 or above must be installed. This is when the [Third-Party Scan REST API](https://help.sonatype.com/iqserver/automating/rest-apis/third-party-scan-rest-api---v2) was incorporated into Nexus IQ Server. 
```

Comment:
Hoping is a fun word, but I'd go with `For users wanting to use Nexus IQ Server as their data source for scanning,`

Reply:
in: fb69e8f